crates/configuration/src/version5/mod.rs

```diff
@@ -51,6 +51,9 @@ pub struct ParsedConfiguration {
     /// Which version of the generated mutation procedures to include in the schema response
     #[serde(default)]
     pub mutations_version: Option<metadata::mutations::MutationsVersion>,
+    /// Provide a custom prefix for generated mutation names. Defaults to mutations version.
```

Comment:
The new config. `serde(default)` means old config that does not include this will get `None`, so we make sure that keeps old behaviour intact.